index.html

```diff
@@ -80,7 +80,156 @@ <h1 class="mn__title">
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__2">
+          <img
+            class="proj__img"
+            src="img/Work 2.webp"
+            alt="2nd project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__3">
+          <img
+            class="proj__img"
+            src="img/Work 3.webp"
+            alt="3rd project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__4">
+          <img class="proj__img" src="img/Work 4.jpg" alt="4th project image" />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__5">
+          <img
+            class="proj__img"
+            src="img/Work 5.webp"
+            alt="5th project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
```

Comment:
[OPTIONAL] I like your design but I think the best way to implement this block is to use a combination  ul tag and li tag.The ul tag  is used for lists.
What do you think about this :arrow_down: 
```
<ul class="proj__lang">
              <li class="bt">Ruby on rails</li>
              <li class="bt">CSS</li>
              <li class="bt">JavaScript</li>
              <li class="bt">HTML</li>
  </ul>
```